coachmark/src/main/java/com/mercadolibre/android/andesui/coachmark/model/AndesWalkthroughCoachmark.kt

```diff
@@ -3,21 +3,30 @@ import androidx.core.widget.NestedScrollView
 import android.view.View
 import java.io.Serializable
 
+data class AndesScrollessWalkthroughCoachmark(
+        var steps: MutableList<AndesWalkthroughCoachmarkStep>,
+        val view: View,
```

Comment:
```suggestion
        val anchorView: View,
```
esta es la vista donde se ancla el walkthrough?

Reply:
Si, exactamente! Le cambio el nombre para que sea mas descriptivo.